conanfile.py

```diff
@@ -0,0 +1,30 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+
+
+class CattalibConan(ConanFile):
+    name = "catta"
+    version = "0.1"
+    license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
+    url = "<Package recipe repository url here, for issues about the package>"
+    description = "<Description of Elearcommonlib here>"
+    topics = ("<Put some tag here>", "<here>", "<and here>")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = "shared=False"
+    generators = "make"
+
+
+    def build(self):
+        autotools = AutoToolsBuildEnvironment(self)
+        self.run("cd .. && autoreconf -fsi ")
+        autotools.configure(configure_dir="..",args=["--prefix=${PWD}"])
+        autotools.make()
+        autotools.install()
+    def package(self):
+        self.copy("*.h", dst="include", src="include/include")
+        self.copy("*.la", dst="lib", src="lib",keep_path=False)
+
+    def package_info(self):
+        self.cpp_info.libs = ["catta"]
```

Comment:
In all conan files, keep a space after "[" and a space before "]" in such declarations

Reply:
handled